Source/monster.h

```diff
@@ -68,6 +68,32 @@ enum MON_MODE {
 	MM_TALK,
 };
 
+enum {

```

Comment:
Can we add the name for this one?
```suggestion
enum monster_action {
```